session_3/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Topic</th>
+      <th>Tags</th>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @articles.each do |article| %>
+      <tr>
+        <td><%= article.title %></td>
+        <td><%= article.topic %></td>
+        <td><%= article.tags %></td>
+        <td><%= article.content %></td>
+        <% begin %>
+          <% if article.id == current_user.id %>
+            <td><%= link_to 'Show', article %></td>
+            <td><%= link_to 'Edit', edit_article_path(article) %></td>
+            <td><%= link_to 'Destroy', article, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+          <% end %>
+        <% rescue => e %>
```

Comment:
That's wrong! Never ignore exceptions!

Reply:
It's not a Error as such that will occur due to user's mistake. It's just that when user is not logged in current_user is set to nil and accessing nil object's id isn't possible and throws error. There's no problem if the User's not logged in, that's why I'm not ignoring it.